### ISSUE TITLE
"notApplicable" required in code list valueUnitOrScale

### ISSUE THREAD

**Issue by nmprista** [ices-tools-dev/RDBES#162]
BVvalueUnitOrScale is Mandatory with code list "ValueUnitOrScale" ([//vocab.ices.dk/?ref=1608](https://vocab.ices.dk/?ref=1608)). 

"notApplicable" does not feature in the list but is required for cases like BVtypeMeasured=="InfoParasite" or BVtypeMeasured=="InfoGenetic".

**Comment by HenrikK-N** [comment]
Yes a code is needed for all the BiologicalMeasurementType, which starts with "Info", the code will be 'NotApplicable'.

**Comment by HenrikK-N** [comment]
"notApplicable" have been added to the code list valueUnitOrScale.
'-9' is used for not available in the code list 'SampleType'-
